import-export-cli/docs/apictl_delete_policy.md

```diff
@@ -0,0 +1,39 @@
+## apictl delete policy
+
+Delete Policy
+
+### Synopsis
+
+Delete a Policy from an environment
+
+```
+apictl delete policy [policy type] (--environment <environment-from-which-the-application-should-be-deleted>) [flags]
+```
+
+### Examples
+
+```
+apictl delete policy api -n addHeader -e dev
+NOTE: Both the flags (--name (-n), and --environment (-e)) are mandatory.
+```
+
+### Options
+
+```
+  -e, --environment string   Environment from which the Policy should be deleted
+  -h, --help                 help for policy
```

Comment:
Is `help for policy` right? Also please make sure to capitalize the start of a sentence. Check in other places too

Reply:
In other docs, they have started with a simple letter for the definition of `-h`. Should we have to change those too?

Reply:
Seems like this is not applicable